its/plugin/plugins/php-custom-rules-plugin/src/main/java/org/sonar/samples/php/CustomPHPVisitorCheck.java

```diff
@@ -23,19 +23,17 @@
 import org.sonar.check.Rule;
 import org.sonar.plugins.php.api.tree.expression.FunctionExpressionTree;
 import org.sonar.plugins.php.api.visitors.PHPVisitorCheck;
-import org.sonar.squidbridge.annotations.SqaleConstantRemediation;
 
 @Rule(
   key = "visitor",
   priority = Priority.MINOR,
   name = "PHP visitor check",
   description = "desc")
-@SqaleConstantRemediation("5min")
 public class CustomPHPVisitorCheck extends PHPVisitorCheck {
 
   @Override
   public void visitFunctionExpression(FunctionExpressionTree tree) {
-    context().newIssue(this, "Function expression.").tree(tree);
+    context().newIssue(this, tree, "Function expression.");
```

Comment:
@pynicolas why this change?

Reply:
`CheckContext.newIssue(PHPCheck check, String message)` is deprecated.